heron/schedulers/src/java/org/apache/heron/scheduler/kubernetes/V1Controller.java

```diff
@@ -519,11 +524,12 @@ private V1StatefulSet createStatefulSet(Resource containerResource, int numberOf
    * @param resource Passed down to configure the resource limits.
    * @param numberOfInstances Passed down to configure the ports.
    * @param isExecutor Flag used to configure components specific to <code>Executor</code> and <code>Manager</code>.
-   * @param configPVC <code>Persistent Volume Claim</code> configurations options.
+   * @param volumes <code>Volumes</code> generated from configurations options.
+   * @param volumeMounts <code>Volume Mounts</code> generated from configurations options.
    */
   private void configurePodSpec(final V1PodTemplateSpec podTemplateSpec, Resource resource,
```

Comment:
Would it be worthwhile to mark this parameters passed in the function as `final`?

Reply:
Changes affected.